incubator/airflow/templates/services.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
     chart: {{ template "airflow.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+  annotations:
```

Comment:
All three of these services use the same annotation (and therefore would have the same service config). Is that generally going to be what we want? Web seems like it faces end users, whereas flower faces admins and worker only faces the web server. Am I right in that understanding? In that case, we may want to configure web differently than the other two, at least.

Reply:
Well, I agree in principle, I was going to match something closer to the current behavior of the current chart, which is that the service `type` is coming from `.Values.airflow.service.type` for both flower and web, though it's a good point about the workers - I should keep those separated for sure.

Maybe I split out the annotations into their respective blocks in the values, at the very least, but I'm also all in on splitting out the `type` as well if you think it's a good idea (so that flower and web aren't forced to be the same type). I'm treading into breaking change territory in that case, though, so that's my worry there.